nemo_skills/inference/prompt/few_shot_examples/examples_proof.py

```diff
@@ -0,0 +1,38 @@
+proof_questions = [
+        {
+        "question": "Prove the identity $a^\\frac{1}{2}-\\frac{a-a^{-2}}{a^\\frac{1}{2}-a^{-\\frac{1}{2}}} +\\frac{1-a^{-2}}{a^\\frac{1}{2}+a^{-\\frac{1}{2}}}+\\frac{2}{a^\\frac{3}{2}}=0$ ",
+        "classification": """
+        Classification: proof_question
+""".strip(),
+    },
+    {
+        "question": "Given a sequence of expressions: $$(a+x)^2;\\ (a^2+x^2);\\ (a-x)^2;\\ldots $$ Prove that they form an arithmetic progression, and find the sum of its first $n$ terms.",
+        "classification": """
+        Classification: proof_question
+""".strip(),
+    },
+    {
+        "question": "Write first several terms of a geometric progression in which the difference between the third and first terms is equal to 9, and that between the fifth and third terms equal 36.",
+        "classification": """
+        Classification: not_proof_question
+""".strip(),
+    },
+    {
+        "question": "Denoting the sums of the first $n_1$, first $n_2$ and first $n_3$ terms of an arithmetic progression by $S_1$, $S_2$ and $S_3$, respectively, show that $$\\frac{S_1}{n_1}(n_2-n_3)+\\frac{S_2}{n_2}(n_3-n_1)+\\frac{S_3}{n_3}(n_1-n_2)=0.$$",
+        "classification": """
+        Classification: proof_question
+""".strip(),
+    },
+    {
+        "question": "Solve the following equations: $\\frac{\\sin(60^\\circ+x)+\\sin(60^\\circ-x)}{2} =\\frac{\\tan x}{(1+\\tan^2 x)^2}+\\frac{\\cot x}{(1+\\cot^2 x)^2}$",
+        "classification": """
```

Comment:
Why couldn't this be "generation": "False"  and "generation": "True" (True for proof_question)

Reply:
Do you think it will work better or just for simplicity? I think it probably does not matter what we use, but potentially we might extend it to more categories in the future (e.g. questions with images), in which case we'd need to use something like what's currently here anyway